ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx

```diff
@@ -85,6 +95,25 @@ export const PodsLogsViewer = (props: {applicationName: string; pod: models.Reso
                             }}>
                             {prefs.appDetails.darkMode ? <i className='fa fa-sun' /> : <i className='fa fa-moon' />}
                         </div>
+                        <div style={{display: 'flex', marginLeft: 'auto'}}>
```

Comment:
The input field looks small on big screen .  Can you increase the width, please?